client/src/components/SponsorsComps/HomeCommunitySponsors.js

```diff
@@ -0,0 +1,106 @@
+/* eslint-disable prettier/prettier */
+import React from 'react';
+import { hyperlink } from "../../utils";
+import { image } from "../../images";
+import styled from "styled-components";
+
+// Styled-component Section
+
+const HomeCommunityContainer = styled.div`
+${ ({ theme: { mq } }) => `
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  
+  @media (max-width: ${mq.s}) {
+    justify-content: center;
+  }  
+`}
+
+`;
+
+const SponsorMiniContainer = styled.div`
+  ${ ({ theme: { space } }) => `
+    flex: 0 0 20%;
+    text-align: center;
+    align-self: center;
+
+    margin: 0 ${space.l};
+  `}
+`;
+
+const SponsorMiniLink = styled.a`
+  min-width: 150px;
+  min-height: 110px;
+
+  position: relative;
+  display: inline-block;
+  
+  cursor: pointer;
+  transition: opacity 300ms linear;
+  
+  &:hover{
+    opacity: 0.5;
+  }
+`;
+
+const SponsorMiniImg = styled.img`
+  max-width: 100%;
+  padding: ${props => props.theme.space.s};
+  max-height: 100px;
+`;
+
+// React components section
+
+const sponsorsOrder = [
+  'Gett', 'Liveperson', 'Oracle', 'Chegg',
+  'Joytunes', 'Trax', 'CyberArk', 'Cloudinary',
+  'Zebra', 'eko', 'Cellebrite', 'Kenshoo',
+  'Bizzabo', 'Wework', 'ThetaRay', 'Tipalti'
+];
+
+const orderSponsors = (sponsors) => {
+  let orderdSponsors = []
+  sponsorsOrder.forEach((key) => {
+    orderdSponsors.push(sponsors.find((sponsor) => sponsor.name === key))
+  })
+  return orderdSponsors
+}
+
+
+class SponsorMini extends React.Component {
+  state = {
+    hovered: false
```

Comment:
I dont see this being used actually 

And if not, lets have a function component and not a class